GameData/RP-0/Tree/ECM-Parts.cfg

```diff
@@ -899,6 +899,7 @@
     ROE-RD200 = RD-200
     ROE-RD211 = RD-211-8D57
     ROE-RD215 = RD-215-8D513
+    ROE-RD254 = RD-254
```

Comment:
Shouldn't this be `ROE-RD254 = RD-254-11D44`? As of now, the part costs nothing to unlock, and the first config does not unlock with the part.